draft-ietf-ppm-dap.md

```diff
@@ -1871,6 +1884,14 @@ The Leader then begins working with the Helper to aggregate the reports
 satisfying the query (or continues this process, depending on the VDAF) as
 described in {{aggregate-flow}}.
 
+Collection job creation MUST be idempotent. That is, multiple requests to
```

Comment:
does it matter if the task has expired and leader deleted it's collection_jobs?

Reply:
Bump

Reply:
I don't understand how task expiration complicates this. We already have this text:

> Upon receipt of a `CollectionReq`, the Leader begins by checking that it
> recognizes the task ID in the request path. If not, it MUST abort with error
> `unrecognizedTask`.

So if task expires, then the leader should refuse to allow new collection jobs to be created in it, regardless of whether the request has been repeated.